README.md

```diff
@@ -1,67 +1,42 @@
 ﻿# .NET Core Home
 
-The core repository is the starting point to engage in and learn about the
-.NET Core stack. 
-
-> **Note:** please note that this repository is not for filing product issues.
-> If you run into an issue using .NET Core, there are multiple repos where you can 
-> file an issue:
-> * [dotnet/cli](https://github.com/dotnet/cli) - for CLI tools and questions
-> * [dotnet/corefx](https://github.com/dotnet/corefx) - for API issues and questions
-> * [dotnet/coreclr](https://github.com/dotnet/coreclr) - for runtime issues
-> * [dotnet/docs](https://github.com/dotnet/docs) - for .NET documentation issues and questions
-> * [dotnet/sdk](https://github.com/dotnet/sdk) - for .NET Core SDK (used by Visual Studio and CLI) issues and questions
-> * [dotnet/standard](https://github.com/dotnet/standard) - for .NET Standard issues and questions
-> * [nuget/home](https://github.com/nuget/home) - for NuGet questions and issues
-> * [aspnet/home](https://github.com/aspnet/home) - for ASP.NET Core questions and issues.
-> * [aspnet/Docs](https://github.com/aspnet/Docs) - for ASP.NET and ASP.NET Core documentation issues and questions
-> * [dotnet/netcorecli-fsc](https://github.com/dotnet/netcorecli-fsc) - for F# questions and issues
-
-> 
-> We will be removing the ability to file issues on this repo in the near future.
+The dotnet/core repository is a good starting point for .NET Core. 
 
 ## Get Started
 
-If you're new to .NET Core and have 10 minutes to try it, start here: 
-- [What is .NET Core?](https://www.microsoft.com/net/core/platform)
-- [Get Started with the Official releases of .NET Core on Windows, OSX and Linux](https://www.microsoft.com/net/core)
+If you're new to .NET Core, you can get started in 10 minutes: 
 
-If you have some more time and want to go deeper or get the latest builds:
-- [Get the latest builds of .NET Core on Windows, OSX and Linux](https://github.com/dotnet/core-setup/blob/master/README.md)
-- [Get the latest builds of ASP.NET Core on Windows, OSX and Linux](https://github.com/aspnet/home)
+- [Get started with .NET Core](https://www.microsoft.com/net/core)
+- [Download .NET Core](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md)
```

Comment:
The problem is that not everyone is looking for the same version. The marketing site doesn't do a good job of catering to that. I also have plans to make the download pages/release notes easier to use.